package.json

```diff
@@ -671,6 +671,12 @@
                         "default": "node",
                         "scope": "application",
                         "description": "The Node.js executable path. for example, C:\\Program Files\\nodejs\\node.exe"
+                    },
+                    "leetcode.colorizeProblems": {
+                        "type": "boolean",
+                        "default": false,
```

Comment:
Let's default to `true`

And also could you please add the new setting into the readme as well? https://github.com/LeetCode-OpenSource/vscode-leetcode#settings
 
Thank you

Reply:
Done.